robottelo/constants.py

```diff
@@ -1758,7 +1758,7 @@
     {'name': 'SmartProxy', 'controller': 'smart_proxies', 'skip_for_ui': True},
     {
         'name': 'ComputeResource', 'controller': 'compute_resources',
-        'setup': entities.DockerComputeResource
+        'setup': entities.DockerComputeResource, 'skip_for_ui': True
```

Comment:
@omkarkhatavkar can't we remove it as docker compute resource is not part of 6.6?

Reply:
Yeah, we can delete that for sure, I was reluctant in start.

Reply:
I don't think removing it completely is good choice, as docker here is used as representation for whole group of compute resources. If we remove it, we lose coverage for all of them. Instead, we should pick another item from the same group of entities to serve as representation.

In other words, change `entities.DockerComputeResource` to `entities.LibvirtComputeResource`. Or any other compute resource. I can't say why docker was picked in the first place.

When providing test results, please modify `tests/foreman/ui/test_bookmark.py::random_entity`, so it always returns compute resource. Due to random nature of tests in that file, subsequent runs may give different results.

Reply:
let me try entities.LibvirtComputeResource, if it is feasible I will work with that otherwise good to skip_for_ui as I don't think it should fail that test based on a single entity.

Reply:
@mirzal no need to update the random_entity, the test itself is failing in `def ui_entities` which is prerequisite for this test case.

Reply:
replacing to entities.LibvirtComputeResource makes more sense and test got passed. I have updated the change in commit `1537bc733fa479c14fc96e715af179ceb23d4769`